docs/building-new-project.md

```diff
@@ -171,7 +166,7 @@ Since that's a bit handful, let me paste the whole content:
   description = "A very basic flake";
 
   inputs = {
-    nixpkgs.url = "github:NixOS/nixpkgs/nixos-23.11";
+    nixpkgs.url = "github:nixos/nixpkgs?ref=nixos-24.05";
```

Comment:
I bet both work, but isn't previous one "easier to type"?

Reply:
yes both should work
The new one is what the default flake from `nix flake init` uses, thats why i chose it.
If the previous style is still preferred I will change it back.